config/routes.rb

```diff
@@ -1,6 +1,21 @@
 Rails.application.routes.draw do
+  devise_for :users
   root 'users#index'
-  resources :users, only: [:index, :show]  do 
-    resources :posts, only: [:index, :show] 
+
+  namespace :api do
+    namespace :v1 do
+      resources :users, only: [] do
+        resources :posts, only: [:index] do
+          resources :comments, only: [:index, :create]
```

Comment:
- [ ] Hi @Ridwanullahi-code great job for making sure that it is possible to get all posts and comments in a JSON format. The only issue now is that I'm unable to add(POST) comment to a post as you can see on the screenshot below:

![Screenshot from 2023-06-24 20-04-46](https://github.com/Ridwanullahi-code/BlogApp/assets/63087491/a97949d7-1f2e-4982-91c8-1841c9497dee)